### ISSUE TITLE
gnt-cluster renew-crypto --new-ssh-keys does not replace the master key

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1052]
Originally reported of Google Code with ID 996.

```
This is a known missing feature for 2.13 and this bug is to keep track of it. Unfortunately this leads to errors if gnt-cluster renew-crypto is used to fix a broken SSH setup. 
```
Originally added on 2014-11-25 10:39:26 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
That was fixed a while ago.
```
Originally added on 2015-04-02 12:09:10 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.14.0
```
Originally added on 2015-06-02 10:37:03 +0000 UTC.

Changed State: Released
